htdocs/accountancy/admin/account.php

```diff
@@ -165,6 +165,7 @@
 if ($db->type == 'pgsql') $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."accounting_account as a2 ON a2.rowid = CAST(aa.account_parent AS INTEGER)";
 else $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."accounting_account as a2 ON a2.rowid = CAST(aa.account_parent AS UNSIGNED)";
 $sql .= " WHERE asy.rowid = " . $pcgver;
+$sql .= " AND aa.entity = " . $conf->entity;
```

Comment:
If you put the AND here, you break the LEFT JOIN.
then AND must be after the ON ... on the LEFT join line.

Reply:
True, It's fix in the new commit